OpenRA.Mods.Common/Traits/Explodes.cs

```diff
@@ -134,6 +134,9 @@ void INotifyDamage.Damaged(Actor self, AttackInfo e)
 			if (Info.DamageThreshold == 0)
 				return;
 
+			if (Info.DeathTypes.Count > 0 && !e.Damage.DamageTypes.Overlaps(Info.DeathTypes))
```

Comment:
More `HashSet<string>` insanity, but looks correct.

Reply:
Come to think of it, aiming Damage/DeathTypers to convert top BitSet in Next1 sounds like a worthwhile aim.